WultraMobileTokenSDK/Operations/Model/Requests/WMTAuthorizationData.swift

```diff
@@ -25,8 +25,45 @@ class WMTAuthorizationData: Codable {
     /// Operation id
     let id: String
     
-    init(operationId: String, operationData: String) {
-        self.data = operationData
-        self.id   = operationId
+    /// Proximity Otp Object
+    let proximityCheck: WMTProximityCheckData?
+    
+    init(operationId: String, operationData: String, proximityCheck: WMTProximityCheckData? = nil) {
+        self.id              = operationId
+        self.data            = operationData
+        self.proximityCheck   = proximityCheck
+    }
+    
+    init(operation: WMTOperation) {
+        self.id            = operation.id
+        self.data          = operation.data
+        
+        guard let proximityCheck = operation.proximityCheck else {
+            self.proximityCheck = nil
+            return
+        }
+        
+        self.proximityCheck = WMTProximityCheckData(
+            totp: proximityCheck.totp,
+            type: proximityCheck.type,
+            timestampRequested: proximityCheck.timestampRequested,
+            timestampSigned: Date()
```

Comment:
It would be good if the timestampSigned would be a parameter of the init with default to `Date()`. We can then use a server time for reference if it's available.